.travis.yml

```diff
@@ -16,10 +16,22 @@ virtualenv:
   system_site_packages: true
 install:
   - pip install -r https-everywhere-checker/requirements.txt
-before_script:
-  - sh -e /etc/init.d/xvfb start
+  - pip install -r test/chromium/requirements.txt
 env:
   - DISPLAY=':99.0'
+before_script:
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
+  - wget https://chromedriver.storage.googleapis.com/2.18/chromedriver_linux64.zip
+  - unzip chromedriver_linux64.zip
+  - sudo mv chromedriver /usr/bin/chromedriver
+  - sudo apt-get install -y python-software-properties
```

Comment:
Try: https://chromedriver.storage.googleapis.com/2.12/chromedriver_linux64.zip

FWIW, I'm just trying to keep this as minimalist as possible -- since in *any* future version of Travis (when they upgrade beyond 12.04 LTS), we should be able to just apt-get install chromium + chromedriver